tesseract_core/runtime/schema_types.py

```diff
@@ -371,3 +373,27 @@ def is_differentiable(obj: Any) -> bool:
 UInt64 = Array[(), "uint64"]
 Complex64 = Array[(), "complex64"]
 Complex128 = Array[(), "complex128"]
+
+
+def _resolve_input_path(path: Path) -> Path:
+    input_path = get_input_path()
+    tess_path = (input_path / path).resolve()
+    if str(input_path) not in str(tess_path):
+        raise ValueError("File references have to be relative to --input-path.")
```

Comment:
I'd add something like "invalid path" or similar before the suggestion written here